lib/ice_cube/parsers/ical_parser.rb

```diff
@@ -31,6 +31,7 @@ def self.rule_from_ical(ical)
 
       ical.split(';').each do |rule|
         (name, value) = rule.split('=')
+        raise ArgumentError, "empty ical rule's value" if value.nil?
```

Comment:
I think the text of this error should be something more like: "Invalid iCal rule component"